recursive_art.py

```diff
@@ -20,8 +20,21 @@ def build_random_function(min_depth, max_depth):
         (See the assignment writ-eup for details on the representation of
         these functions)
     """
-    # TODO: implement this
-    pass
+    function = ['prod','avg','cos_pi','sin_pi','tan_pi','square']
+
+    if(min_depth < 1):
+        if(random.randint(0, 1) == 1):
+            return ['x']
+        return ['y']
+    if(max_depth == 0):
+        if(random.randint(0, 1) == 1):
+            return ['x']
+        return ['y']
+    choice = random.randint(0, len(function) - 1)
+    if(choice > 4):
```

Comment:
it would be easier to understand the code if you put an inline comment explaining why you made this choice.